tests/unit_tests/test_tethys_apps/test_models/test_PersistentStoreDatabaseSetting.py

```diff
@@ -20,16 +20,12 @@ class PersistentStoreDatabaseSettingTests(TethysTestCase):
     def set_up(self):
         self.test_app = TethysApp.objects.get(package="test_app")
 
-        # Get default database connection if there is one
-        if "default" in settings.DATABASES:
-            self.conn = settings.DATABASES["default"]
-        else:
-            self.conn = {
-                "USER": "tethys_super",
-                "PASSWORD": "pass",
-                "HOST": "localhost",
-                "PORT": "5435",
-            }
+        self.conn = {
+            "USER": "tethys_super",
+            "PASSWORD": "pass",
+            "HOST": "localhost",
+            "PORT": "5435",
+        }
```

Comment:
Why this change?

Reply:
I'm not sure why it was using the actual database configuration for these tests in the first place. It seems like for the testing environment we want to control the inputs. 

In any case, when using sqlite then the database configuration won't have any this information, so getting the actual database config will cause the tests to fail.